worker/streak.js

```diff
@@ -106,5 +109,16 @@ export function checkStreak ({ models }) {
       FROM streak_started`
 
     console.log('done checking streak', id)
+
+    if (!affected) return
+
+    // new streak started for user
+    const index = Math.floor(Math.random() * FOUND_BLURBS.length)
+    const blurb = FOUND_BLURBS[index]
+    sendUserNotification(id, {
+      title: 'you found a cowboy hat',
+      body: blurb,
+      tag: 'STREAK'
+    })
```

Comment:
One thing to note here is that the blurb will be different in the push notification to the notification in /notifications. In /notification, it uses `itemId % 6` and here I just use a random index since I don't have access to the item id afaict.

Reply:
Perhaps we can randomize the blurb on some shared data with the notification that you do have access to? Maybe the millisecond or something?

Reply:
lol, I _thought_ the frontend uses `itemId` but `n.id` is the streak id. And obviously, I have access to it in this context